generator/integration_tests/golden/iam_credentials_metadata_decorator_test.cc

```diff
@@ -105,6 +110,27 @@ TEST_F(MetadataDecoratorTest, GenerateIdToken) {
   EXPECT_EQ(TransientError(), status.status());
 }
 
+TEST_F(MetadataDecoratorTest, WriteLogEntries) {
+  EXPECT_CALL(*mock_, WriteLogEntries(_, _))
+      .WillOnce(
+          [this](grpc::ClientContext &context,
+                 google::test::admin::database::v1::WriteLogEntriesRequest const
+                     &) {
```

Comment:
Here too. Not sure if these are typos, or clang-format did this, or if they were generated like this. :shrug: